### PR TITLE
create login page

### PR DESCRIPTION
**Pull request by devsurajrai**
Please review this PR. 
Things to review:
- Login Form
- Login Component

Screenshot 2022-03-15 at 6 51 18 PM

### CODE REVIEW

**Comment by sushmita12345** [comment]
1. Clean code
2. Naming convention is good

**Review by sushmita12345 — APPROVED** [review]

File: src/components/LoginMain.component/LoginMain.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          <input
+            className="input-simple br-l m-b-s"
+            type="password"
+            placeholder="Enter Password"
+            required=""
+          />
+        </label>
+
+        <label className=" text-sm lh-md login-label flex-r flex-center" for="">
+          <input className="input-checkbox m-r-s" type="checkbox" />{" "}
```
Comment (Dev-AkshatGupta) [inline src/components/LoginMain.component/LoginMain.jsx]: Please remove empty jsx elements

File: src/components/header.component/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  <header className="nav-header  nav-header--e-comm nav--e-comm-bg  w-p-full">
+    <nav className="nav nav--e-comm  nav--e-comm-bg flex-r flex-s flex-sp-eve ">
+      {/* <!-- Brand Name  --> */}
+      <a
+        className=" brand-name--e-comm text-md m-lr-md "
+        href="../../index.html"
+      >
+        PHOENIX<sub>SHOP</sub>
+      </a>
+      {/* <!-- Search Field  --> */}
```
Comment (Dev-AkshatGupta) [inline src/components/header.component/Header.jsx:12]: Nice commenting This increases the developers experience

**Review by Dev-AkshatGupta — APPROVED** [review]
This is nice work Code looks appropriate to me
And I wanna congratulate you for  your work regarding the comments you have written for developer experience .

File: src/components/header.component/Header.jsx (hunk 2 above)
Comment (devsurajrai) [reply src/components/header.component/Header.jsx:12]: Done

**Pull request merged**